integration-cli/docker_cli_swarm_test.go

```diff
@@ -27,6 +27,7 @@ import (
 	remoteipam "github.com/docker/docker/libnetwork/ipams/remote/api"
 	"github.com/docker/docker/pkg/plugins"
 	"github.com/docker/docker/testutil"
+	tudaemon "github.com/docker/docker/testutil/daemon"
```

Comment:
Could you use `testdaemon` as alias for this one? It looks like that's the alias we converged on (I should probably add a check for that in golang-ci)

```bash
git grep ' "github.com/docker/docker/testutil/daemon"'
integration-cli/check_test.go:  testdaemon "github.com/docker/docker/testutil/daemon"
integration-cli/docker_api_swarm_service_test.go:       testdaemon "github.com/docker/docker/testutil/daemon"
integration-cli/docker_api_swarm_test.go:       testdaemon "github.com/docker/docker/testutil/daemon"
integration-cli/docker_cli_daemon_test.go:      testdaemon "github.com/docker/docker/testutil/daemon"
integration-cli/docker_cli_external_volume_driver_test.go:      testdaemon "github.com/docker/docker/testutil/daemon"
integration-cli/docker_cli_logs_test.go:        testdaemon "github.com/docker/docker/testutil/daemon"
integration-cli/docker_cli_network_unix_test.go:        testdaemon "github.com/docker/docker/testutil/daemon"
integration-cli/docker_cli_run_test.go: testdaemon "github.com/docker/docker/testutil/daemon"
integration-cli/docker_hub_pull_suite_test.go:  testdaemon "github.com/docker/docker/testutil/daemon"
```